### DISCUSSION TITLE
Led on when press keyboard key and off when i press the key again.

### DISCUSSION THREAD

**Discussion by Basiliadis** [tttapa/Control-Surface#1044] [General]
Hello all,

how is it possible to led on/ff when press a keyboard key?

thank you very much.

**Comment by tttapa** [comment]
How did you connect the keyboard to the Arduino? Is the Arduino scanning the keyboard keys, or do they communicate over MIDI?

**Reply by Basiliadis** [reply]
can control surface use keyboard keys scan arduino code?.. i think that it will  very useful!

**Comment by Basiliadis** [comment]
Hello again....how can add code that arduino can read which keyboard key or keyboard keys combination is press please?
